apps/genetics/src/utils/variant.ts

```diff
@@ -1,12 +1,17 @@
-export function variantHasInfo(data) {
-  return data && data.variantInfo;
-}
-export function variantGetInfo(data) {
-  return data.variantInfo;
-}
```

Comment:
These functions are unused in the codebase, and overly basic so they can be re-added later if necessary.